library/entity/multipart/src/main/java/org/quiltmc/qsl/entity/multipart/api/AbstractEntityPart.java

```diff
@@ -181,29 +181,43 @@ public void move(double dx, double dy, double dz) {
 	 * Rotates this {@link AbstractEntityPart} about the pivot point with the given rotation.
 	 *
 	 * @param pivot the pivot point to rotate about in relative coordinates
-	 * @param pitch the rotation about z-axis in degrees
+	 * @param pitch the rotation about x-axis in degrees
 	 * @param yaw   the rotation about y-axis in degrees
```

Comment:
Javadoc is inaccurate.

```suggestion
	 * @param pitch   the rotation about x-axis
	 * @param yaw     the rotation about y-axis
	 * @param degrees whether the rotation should be done in degrees or radians
```